adele-application/src/main/java/com/microservicesteam/adele/configuration/AdeleConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package com.microservicesteam.adele.configuration;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+import java.util.function.Supplier;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AdeleConfiguration {
+
+    @Bean
+    public Supplier<String> uuidGenerator() {
+        return () -> UUID.randomUUID().toString();
```

Comment:
Did you change your mind about using UUID instead of String?

Reply:
I wanted to stay coherent with the current implementation. I think we should change all of them in one pull request.